go/client/logclient.go

```diff
@@ -386,3 +403,28 @@ func (c *LogClient) GetEntries(start, end int64) ([]ct.LogEntry, error) {
 	}
 	return entries, nil
 }
+
+// GetProofByHash returns an audit path for the hash of an SCT.
+func (c *LogClient) GetProofByHash(hash []byte, treeSize uint64) (ct.AuditPath, error) {
+	var resp getProofByHashResponse
+	b64Hash := url.QueryEscape(base64.StdEncoding.EncodeToString(hash))
+	u := fmt.Sprintf("%s%s?tree_size=%d&hash=%v", c.uri, GetProofByHashPath, treeSize, b64Hash)
+	log.Printf("%v", u)
+	err := c.fetchAndParse(u, &resp)
+	if err != nil {
+		return nil, err
+	}
+	if resp.ErrorMessage != "" {
```

Comment:
I think you might be disappointed by some recent developments in `fetchAndParse` (and might need a further rebase, at the minimum)...